partner_communication_switzerland/models/partner_communication.py

```diff
@@ -52,6 +52,19 @@ class PartnerCommunication(models.Model):
         readonly=False,
     )
 
+    def print_letter(self, print_name, **print_options):
+        """
+        Adds duplex printing option for Konica Minolta depending on page count.
+        """
+        if len(self) > 1:
+            page_counts = list(set(self.mapped("pdf_page_count")))
+            # Duplex if all documents have a pair page count
+            if len(page_counts) == 1 and page_counts[0] % 2 == 0:
+                print_options["KMDuplex"] = "2Sided"
+            else:
+                print_options["KMDuplex"] = "1Sided"
```

Comment:
Wouldn't that mean that for print option to be set to "2sided" all communication should be of the same length ?

This should work for different len of communication as long as they are all of even length
```suggestion
            sided_option = "2sided"
            for p_count in page_counts: 
                if (p_count % 2 != 0):
                    sided_option = "1Sided"
                    break
            print_options["KMDuplex"] = sided_option

```